README.md

```diff
@@ -17,20 +17,24 @@ A media server for all the many media resources at RAW 1251AM.
 ## Installation (Production)
 
 1. Install [Docker CE](https://docs.docker.com/install/).
-2. `docker run --publish 8080:8080 --env UPLOAD_USER=<insert user> --env UPLOAD_PASSWORD=<insert password> --detach --name media raw1251am/media-server:latest`
-3. Visit server on port `8080`.
+2. Run `docker volume create media-vol` to create a new persistent local volume to store our static assets. You only need to do this once per host.
```

Comment:
I'm not sure if this is the right solution to this. I think that a straightforward volume mapping is a better solution, as that'll work with our existing setup without needing any changes. e.g. `docker run ... -v "/src/folder:/www/media/static" ... raw1251am/media-server:latest`